src/styles/blocks/calendar.scss

```
@@ -0,0 +1,52 @@
+@function week-width($day-width, $gap-between-days) {
+  @return $day-width * 7 + $gap-between-days * 6 + 20px;
+}
+
+$week-days: mon, tue, wed, thu, fri, sat, sun;
```

If we want to use the loop approach, this needs to become:
```suggestion
$week-days:
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
```